docs/extras/modules/model_io/models/llms/integrations/llama_cpp.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_class_name: node-only
+---
+
+# Llama CPP
+
+:::tip Compatibility
+Only available on Node.js.
+:::
+
+This based on the [node-llama-cpp](https://github.com/withcatai/node-llama-cpp) Node.js bindings for [llama.cpp](https://github.com/ggerganov/llama.cpp), allowing you to work with a locally running LLM.
+
+## Setup
+
+You'll need to install the [node-llama-cpp](https://github.com/withcatai/node-llama-cpp) module to communicate with you local model.
+
+```bash npm2yarn
+npm install -S node-llama-cpp
+```
+
+You will also need a local Llama 2 model (or a model supported by [node-llama-cpp](https://github.com/withcatai/node-llama-cpp)). You will need to pass the path to this model to the LlamaCpp module as a part of the parameters (see example).
```

Comment:
Might be good to add more information on how to get a local model set up here? I personally wasn't able to.

Reply:
Great idea, I'll add some docs on the steps I took to get this running.